sdk/identity/identity/samples/tsconfig.json

```diff
@@ -1,17 +1,9 @@
 {
-    "extends": "../tsconfig.json",
-    "compilerOptions": {
-      "module": "commonjs",
-      "declaration": false,
-      "declarationMap": false,
-      "sourceMap": false,
-      "outDir": "../"
-    },
-    "include": [
-      "**/*.ts"
-    ],
-    "exclude": [
-      "src/**/*",
-      "test/**/*"
-    ]
+  "extends": "../tsconfig.samples.json",
```

Comment:
Where does this file come from? I don't see it checked in to master or in this PR.

Reply:
Sorry, I just added the file